rpi_backlight/__init__.py

```diff
@@ -161,12 +198,22 @@ def power(self) -> bool:
         :type: bool
         """
         # 0 is on, 1 is off
```

Comment:
Move this back inside the condition please as it's specific to `return not self._get_value("bl_power")` (explains the inversion)